Js.js

```diff
@@ -0,0 +1,26 @@
+$(document).ready(function(){
+
+//HAD TO TIME BOX THIS ONE, WAS GETTING ERRORS ON MY RETURN STATEMENT, I THINK MY ORDER IS WAY OFF HERE, GONNA TACKLE THE NEXT ASSIGNMENT AND COME BACK TO IT TIME WILLING.
+
+//set a variable time, then write function to make it countdown.
+  var time = 0;
+  var timer= setInterval(timer, 1000);
```

Comment:
I don't think you can have a `setInterval` call itself, as you're doing here. Usually it looks something like this:

```js
setInterval(oneSec, 1000);
function oneSec(){
  alert("A second passed!");
}
```